sql/20201012.sql

```diff
@@ -0,0 +1,36 @@
+--- Table to explicit track thread state
+
+CREATE TABLE threads (
+    slack_parent_message_ts text,
```

Comment:
You should add indexes to all of these columns (except maybe user_phone_number?)